packages/pegasus/bundles/install-on-chain.js

```diff
@@ -0,0 +1,76 @@
+// @ts-check
+import { E } from '@agoric/eventual-send';
+
+import pegasusBundle from './bundle-pegasus';
+
+/**
+ * @param {Object} param0
+ * @param {ERef<NameHub>} param0.agoricNames
+ * @param {ERef<Board>} param0.board
+ * @param {Store<NameHub, NameAdmin>} param0.nameAdmins
+ * @param {NameHub} param0.namesByAddress
+ * @param {ERef<ZoeService>} param0.zoe
+ */
+export async function installOnChain({ agoricNames, board, nameAdmins, namesByAddress, zoe }) {
+  // Fetch the nameAdmins we need.
+  const [installAdmin, instanceAdmin, uiConfigAdmin] = await Promise.all(
+    ['installation', 'instance', 'uiConfig'].map(async edge => {
+      const hub = /** @type {NameHub} */ (await E(agoricNames).lookup(edge));
+      return nameAdmins.get(hub);
+    }),
+  );
+
+  /** @type {Array<[string, SourceBundle]>} */
+  const nameBundles = [['pegasus', pegasusBundle]];
+  const [pegasusInstall] = await Promise.all(
+    nameBundles.map(async ([name, bundle]) => {
+      // Install the bundle in Zoe.
+      const install = await E(zoe).install(bundle);
+      // Advertise the installation in agoricNames.
+      await E(installAdmin).update(name, install);
+      // Return for variable assignment.
+      return install;
+    }),
+  );
+
+  const terms = harden({
+    board,
+    namesByAddress,
+  });
+
+  const { instance, creatorFacet } = await E(zoe).startInstance(pegasusInstall, undefined, terms);
+ 
+  const pegasusUiDefaults = {
+    CONTRACT_NAME: 'Pegasus',
+    BRIDGE_URL: 'http://127.0.0.1:8000',
```

Comment:
This reminds mthat links to localhost conflict with some policy in Brave. I don't suppose any change here is called for, though.
https://github.com/Agoric/agoric-sdk/issues/2730